lib/services/image_service.dart

```diff
@@ -0,0 +1,82 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:image_cropper/image_cropper.dart';
+import 'package:talawa/locator.dart';
+
+/// ImageService class provides different functions as service in the context of Images.
+///
+/// Services include:
+/// * `cropImage`
+/// * `convertToBase64`
+class ImageService {
+  /// Global instance of ImageCropper.
+  final ImageCropper _imageCropper = locator<ImageCropper>();
+
+  /// Crops the image selected by the user.
+  ///
+  /// Accepts a `File` type image and returns a `Future<File?>` of the cropped image.
```

Comment:
This is redundant. Param and return are already written below. If there is nothing needed more than the first line, it can be removed. Otherwise, write something useful.